app/controllers/tasks_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
       redirect_to tasks_url
     else
       flash.now[:danger] = I18n.t 'task_create_failed'
-      render 'new'
+      render turbo_stream: turbo_stream.update('flash', partial: 'shared/error_messages')
```

Comment:
```suggestion
      render :new, status: :unprocessable_entity
```
これで `flash.now` が動いたので `.erb` とか変えずにrenderだけこれにしちゃうのが良いかも！

Reply:
これですね !
https://qiita.com/P-man_Brown/items/862503a638801fea01e7

修正後、動作確認できました！